packages/core/src/build/configAndIndexingFunctions/configAndIndexingFunctions.ts

```diff
@@ -11,16 +11,33 @@ import {
   getRpcUrlsForClient,
   isRpcUrlPublic,
 } from "@/config/networks.js";
-import type { Factory, LogFilter, Source } from "@/config/sources.js";
+import type { Factory, LogFilter, Source, Topics } from "@/config/sources.js";
 import { chains } from "@/utils/chains.js";
 import { toLowerCase } from "@/utils/lowercase.js";
+import { dedupe } from "@ponder/common";
 import parse from "pg-connection-string";
-import type { LogTopic } from "viem";
+import type { Hex } from "viem";
 
-export async function buildConfig({
+export type RawIndexingFunctions = {
+  name: string;
+  fn: (...args: any) => any;
+}[];
+
+export type IndexingFunctions = {
+  [sourceName: string]: {
+    [eventName: string]: (...args: any) => any;
+  };
+};
+
+export async function buildConfigAndIndexingFunctions({
   config,
+  rawIndexingFunctions,
   options,
-}: { config: Config; options: Pick<Options, "ponderDir" | "rootDir"> }) {
+}: {
+  config: Config;
+  rawIndexingFunctions: RawIndexingFunctions;
```

Comment:
I guess the distinction here is that there is no "raw" version of a config the way that there is with indexing functions because the config is built into "sources" and "networks"

Reply:
Got it, makes sense!